src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -2548,6 +2548,9 @@ public static partial class Math
         public static double Atan2(double y, double x) { throw null; }
         public static double Atanh(double d) { throw null; }
         public static long BigMul(int a, int b) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+	    public static ulong BigMul(ulong a, ulong b, out ulong low) { throw null; }
```

Comment:
nit: four spaces instead of a tab
```suggestion
        public static ulong BigMul(ulong a, ulong b, out ulong low) { throw null; }
```